support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -262,7 +264,10 @@ export function ThankYouComponent({
 	): ThankYouModuleType[] => (condition ? [moduleType] : []);
 
 	const thankYouModules: ThankYouModuleType[] = [
-		...maybeThankYouModule(isNotRegistered && !isGuardianAdLite, 'signUp'), // Complete your Guardian account
+		...maybeThankYouModule(
```

Comment:
Presuming this would not be available if still pending

Reply:
Is that because we create an account for them during the backend process, which may not be done? That seems like a reasonable assumption 👍